src/systems/systems/src/simulation/update_context.rs

```diff
@@ -12,6 +15,10 @@ pub struct UpdateContext {
     pub indicated_altitude: Length,
     pub ambient_temperature: ThermodynamicTemperature,
     pub is_on_ground: bool,
+    pub acc_long: Acceleration,
+    pub wheel_center_rpm: f64,
+    pub wheel_left_rpm: f64,
+    pub wheel_right_rpm: f64,
```

Comment:
**issue**: The wheels are not a concern "external to the aircraft".
**suggestion**: Move these out of the `UpdateContext`.